.github/workflows/docker-test.yml

```diff
@@ -14,15 +14,30 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Start containers
+      - name: Install requirements
+        # make sure nvme-cli installed (we need it for /etc/nvme/hostnqn and /etc/nvme/hostid)
+        run: sudo apt-get install -y nvme-cli
+
+      - name: Load Kernel drivers
+        run: sudo modprobe -v nvme-fabrics
+
+      - name: Fix D-BUS
+        run: |
+          sed 's/@STAFD_DBUS_NAME@/org.nvmexpress.staf/g' etc/dbus-1/system.d/org.nvmexpress.staf.in.conf | sudo tee /usr/share/dbus-1/system.d/org.nvmexpress.staf.conf
```

Comment:
I'm not sure why this is needed. The installed D-Bus conf files should not have any parameters like `@STAFD_DBUS_NAME@` in them?

Reply:
the files installed inside the container but we need them on the host... so either this or copy from inside container to the host... this is better i think